services/core-web/src/components/noticeOfWork/NOWActionWrapper.js

```diff
@@ -1,15 +1,28 @@
+/* eslint-disable */
```

Comment:
Remove and fix issues?

Reply:
Removed in the Followup PR - this section is still not fully implemented